include/unifex/when_all.hpp

```diff
@@ -194,9 +194,10 @@ struct _op<Receiver, Senders...>::type {
   template<std::size_t Index, typename Operation>
   friend class _element_receiver;
 
-  explicit type(Receiver&& receiver, Senders&&... senders)
+  template<typename... Senders2>
+  explicit type(Receiver&& receiver, Senders2&&... senders)
     : receiver_((Receiver &&) receiver),
-      ops_(*this, (Senders &&) senders...) {}
+      ops_(*this, (Senders2 &&) senders...) {}
```

Comment:
The senders should be perfectly forwarded through here so I wouldn't have thought that the Senders2 template parameter would have been necessary.
What problems were you seeing with this?

Reply:
I reverted this and it compiles locally. Thanks!